src/include/lean/lean.h

```diff
@@ -2064,11 +2066,11 @@ static inline uint8_t lean_version_get_is_release(lean_obj_arg _unit) {
 }
 
 static inline lean_obj_res lean_version_get_special_desc(lean_obj_arg _unit) {
-    return lean_mk_string(LEAN_SPECIAL_VERSION_DESC);
+    return lean_mk_ascii_string(LEAN_SPECIAL_VERSION_DESC);
```

Comment:
I’d probably lean towards using `mk_utf8_string` here. It’s probably not performance critical, and maybe some German developer with a ü in their name builds lean from a branch with `v0.7.8-Günnar`….
```suggestion
    return lean_mk_utf8_string(LEAN_SPECIAL_VERSION_DESC);
```